games-emulation/mupen64plus_libretro/mupen64plus_libretro-2.5_20171119.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A port of Mupen64 plus, a Nintendo 64 emulator to the libretro API"
+DESCRIPTION="Mupen64Plus is a cross-platform plugin-based N64 emulator which \
+is capable of accurately playing many games. This is the libretro core."
+HOMEPAGE="https://mupen64plus.org/"
+COPYRIGHT="2001-2018 Hacktarux, richard42g, the libretro team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="6f80cbc13978a192193c828010e894a172b9d917"
+SOURCE_URI="https://github.com/libretro/mupen64plus-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1b1dc6d242ae1954dcef75d7dc870edf614fc2788e80a7c7065238a4ae5460b6"
+SOURCE_FILENAME="mupen64plus-libretro-${portVersion/_/-}-$srcGitRev.tar.gz"
+SOURCE_DIR="mupen64plus-libretro-$srcGitRev"
+ADDITIONAL_FILES="mupen64plus_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	mupen64plus_libretro$secondaryArchSuffix = $portVersion
+	addon:mupen64plus_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
```

Comment:
```diff
-	devel:libpng$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
```

Reply:
Since everything else is OK, I can take care of the 2 `libpng` -> `libpng16` substitutions if you agree.